apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -52,26 +53,80 @@ class AnimationPickerBody extends React.Component {
     spriteLab: PropTypes.bool
   };
 
-  state = {
-    searchQuery: '',
-    categoryQuery: '',
-    currentPage: 0
+  constructor(props) {
+    super(props);
+
+    const initialState = {
+      searchQuery: '',
+      categoryQuery: '',
+      currentPage: 0,
+      libraryManifest: props.spriteLab ? spriteCostumeLibrary : animationLibrary
+    };
+    const results = this.searchAssets(initialState.currentPage, initialState);
+    this.state = {
+      ...initialState,
+      results
+    };
+  }
+
+  searchAssets = (page, config = {}) => {
```

Comment:
This looks like a recursive call. I did a double take :) Since this is a wrapper for searchAssets, name it differently. i.e. searchAssetsWrapper or searchAssetsWithDefaults or something similar.